jpo-ode-plugins/src/main/java/us/dot/its/jpo/ode/plugin/ServiceRequest.java

```diff
@@ -101,6 +101,15 @@ public boolean equals(Object obj) {
    private SituationDataWarehouse.SDW sdw;
    private RSU[] rsus;
    private SNMP snmp;
+   private String uuid;
```

Comment:
Any downside to having this as a UUID instead of String?

Reply:
I originally added that as a String so that the SerDes class could support it for the k-table without being overridden. This uuid field is no longer present in the ServiceRequest class as there is a unique streamId already present on the TIM that I'm using to identify it instead.